src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static final int NUM_OF_FIGURES = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[NUM_OF_FIGURES];
+

```

Comment:
redundant blank line

Reply:
sorry, it was for preventing being confused